src/main/java/com/openlattice/chronicle/services/ChronicleServiceImpl.java

```diff
@@ -58,6 +61,8 @@
 import org.slf4j.LoggerFactory;
 import org.springframework.scheduling.annotation.Scheduled;
 
+import com.openlattice.chronicle.data.DeleteType;
```

Comment:
This import appears to be present in two places.
There's an IntelliJ setting here that should prevent this from ever happening once this box is checked:
<img width="483" alt="Screenshot 2020-06-09 12 14 52" src="https://user-images.githubusercontent.com/336506/84189974-21476500-aa64-11ea-9d32-e05257936fec.png">